components/common/SocialLinks/SocialLinks.tsx

```diff
@@ -4,11 +4,11 @@ import {BsFacebook, BsGithub, BsInstagram, BsYoutube} from "react-icons/bs";
 import {FaDiscord, FaLinkedinIn} from "react-icons/fa";
 import styles from "../../../styles/components/SocialLinks.module.scss"
 
-export const SocialLinks = () => {
+export const SocialLinks = ({branch} : {branch: string}) => {
```

Comment:
make this prop optional, it is breaking at dashboard level.

> We dont need branch to be mandatory field.

Reply:
on my way